apl-db-updater/src/main/resources/db/migration/h2/apl/V1_0__Initialize_tables.sql

```diff
@@ -1136,40 +1085,38 @@ CREATE TABLE IF NOT EXISTS `tag`
 
 CREATE TABLE IF NOT EXISTS `tagged_data`
 (
-    `db_id`               bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+    `db_id`               bigint(20) unsigned NOT NULL AUTO_INCREMENT primary key,
     `id`                  bigint(20) NOT NULL,
     `account_id`          bigint(20) NOT NULL,
-    `name`                varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
-    `description`         varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-    `tags`                varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
-    `parsed_tags`         longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin DEFAULT NULL CHECK (json_valid(`parsed_tags`)),
-    `type`                varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+    `name`                varchar(255)  NOT NULL,
+    `description`         varchar(255)  DEFAULT NULL,
+    `tags`                varchar(255)  DEFAULT NULL,
+    `parsed_tags`         longtext DEFAULT NULL,
+    `type`                varchar(255)  DEFAULT NULL,
     `data`                blob       NOT NULL,
     `is_text`             tinyint(1) NOT NULL,
-  `channel` varchar(255) COLLATE utf8_bin DEFAULT NULL,
-  `filename` varchar(255) COLLATE utf8mb4_unicode_ci DEFAULT NULL,
+  `channel` varchar(191)  DEFAULT NULL,
+  `filename` varchar(255)  DEFAULT NULL,
   `block_timestamp` int(11) NOT NULL,
   `transaction_timestamp` int(11) NOT NULL,
   `height` int(11) NOT NULL,
   `latest` tinyint(1) NOT NULL DEFAULT 1,
-  UNIQUE KEY `db_id` (`db_id`),
   UNIQUE KEY `tagged_data_id_height_idx` (`id`,`height`),
   KEY `tagged_data_expiration_idx` (`transaction_timestamp`),
   KEY `tagged_data_account_id_height_idx` (`account_id`,`height`),
   KEY `tagged_data_block_timestamp_height_db_id_idx` (`block_timestamp`,`height`,`db_id`),
-  KEY `tagged_data_channel_idx` (`channel`(191),`height`)
+  KEY `tagged_data_channel_idx` (`channel`,`height`)
```

Comment:
Index 'tagged_data_channel_idx' will not work on MariaDb because of removed COLLATE utf8_bin